site/templates/collection/carousel/item/title.jet

```diff
@@ -0,0 +1,3 @@
+{{block carouselItemTitle(title)}}
+  <h3>{{title}}</h3>
```

Comment:
Is it worth having a separate jet for this?

Reply:
The thinking behind separating this out is to minimise the impact of customisations when some clients inevitably complain that the release year has been removed and want it put back in.  Probably can just handle that with CSS though.